### DISCUSSION TITLE
Support security keys (ed25519-sk & ecdsa-sk)

### DISCUSSION THREAD

**Discussion by sachaw** [Vinzent03/obsidian-git#621] [Problems]
OpenSSH 8.2 onwards supports two new key types that support FIDO2 authenticators, much like a normal password protected private key, a pin is required followed by the user interacting with the physical token.

Is it feasible with the current plugin to add support for this?

Thanks.

**Comment by Vinzent03** [comment]
I don't know these new features, but on desktop, authentication is done by the system and not really by the plugin itself, so it should work without any changes.

**Reply by DerRockWolf** [reply]
Can confirm, on desktop it works out of the box.
